sync/handler.go

```diff
@@ -83,8 +83,8 @@ func newTxsRequestHandler(s *Syncer, logger log.Log) func(msg []byte) []byte {
 		txs, missinDB := s.GetTransactions(txids)
 
 		for _, t := range missinDB {
-			if tx := s.txpool.Get(t); tx != nil {
-				txs[t] = tx.(*types.AddressableSignedTransaction)
+			if tx, err := s.txpool.Get(t); err == nil {
```

Comment:
why not `tx, err := txpool.Get(); if err != nil { return err }`

Reply:
handlers do not return errors, I wanted to use the if one liner